app.js

```diff
@@ -4,12 +4,13 @@ const axios = require("axios");
 const fs = require("fs");
 
 // import modal blocks
+const modals = require("./blocks/modals");
 const {
   launch_modal,
   approval_details,
   quote_lines_details,
   deal_stats,
-} = require("./blocks/modals");
+} = modals;
```

Comment:
Could we deconstruct the `modals` object in line 7 please? I see you did it for the other imports and it would be nice to keep it consistent. 😄

Reply:
@clrmachado I need to keep it like this because I reference `modals` and the items within `modals` (e.g. `approver_details`) separately 🙂

Reply:
ohh I see! I missed that! We can disregard this comment then!